### PR TITLE
Example of gateway operating in a docker compose environment

### PR DESCRIPTION
**Pull request by suessflorian**
Closes https://github.com/movio/bramble/issues/48

### CODE REVIEW

File: examples/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - 8080
+  nodejs-server:
+    build:
+      context: nodejs-service
+    expose:
+      - 8080
+  gateway:
+    image: ghcr.io/movio/bramble
+    volumes:
+      - ${PWD}/example-config.json:/config.json
```
Comment (pkqk) [inline examples/docker-compose.yaml:21]: Where does the `PWD` value come from? Are you using a `.env` file that's not commited?
```suggestion
      - ./example-config.json:/config.json
```

File: examples/gqlgen-service/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM golang
+
+WORKDIR /go/src/app
+
+COPY . .
+RUN go mod init gqlen-server
```
Comment (lucianjon) [inline examples/gqlgen-service/Dockerfile:6]: @suessflorian I'd make these submodules of bramble (github.com/movio/bramble/examples/gqlgen-server) and instantiate them outside of docker. Just so the behaviour and dependencies are the same when running inside of docker and out.

Comment (pkqk) [reply examples/gqlgen-service/Dockerfile:6]: That will make any dependencies in the example services be dependencies of `bramble`, even if they are not necessary?

Comment (pkqk) [reply examples/gqlgen-service/Dockerfile:6]: I would do the build outside of the docker image if possible, the cache interactions with doing the build in the image slow things down a lot if you are changing the code.

Comment (lucianjon) [reply examples/gqlgen-service/Dockerfile:6]: Isn't that the case now?  These examples are going up the tree to bramble's `go.mod` and so their dependencies get injected. This should do the reverse of that.

If I do this:

```
cd examples/gqlgen-service && go mod init github.com/movio/bramble/examples/gqlgen && go mod tidy
cd ../../ && go mod tidy
```

It drops dependencies from bramble's main `go.mod` file.

Comment (lucianjon) [reply examples/gqlgen-service/Dockerfile:6]: By the way, by currently I mean before this PR. I just think we should move the `go mod init` to outside docker and commit the `go.mod` and `go.sum` files.

File: examples/gqlgen-service/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM golang
+
+WORKDIR /go/src/app
+
+COPY . .
+
+RUN go get
```
Comment (lucianjon) [inline examples/gqlgen-service/Dockerfile:7]: @suessflorian Think you're missing a `go generate` somewhere here.

**Pull request merged**